packages/build/tests/monitor/tests.js

```diff
@@ -1,120 +1,191 @@
 import { platform } from 'process'
 
+import { Fixture, normalizeOutput } from '@netlify/testing'
 import test from 'ava'
 import hasAnsi from 'has-ansi'
 import sinon from 'sinon'
 
+// TODO: this need to be done otherwise sinon cannot stub the @netlify/build package due to npm workspace symlinks
+// maybe we can change this if we have a better stubbing library with esm support.
+// Thx to Daniel Tschinder :D
+import { zipItAndShipIt } from '../../../testing/node_modules/@netlify/build/lib/plugins_core/functions/index.js'
 import { CUSTOM_ERROR_KEY } from '../../lib/error/info.js'
-import { zipItAndShipIt } from '../../lib/plugins_core/functions/index.js'
-import { runFixture } from '../helpers/main.js'
 
 const BUGSNAG_TEST_KEY = '00000000000000000000000000000000'
-const flags = { testOpts: { errorMonitor: true }, bugsnagKey: BUGSNAG_TEST_KEY }
 
 test('Report build.command failure', async (t) => {
-  await runFixture(t, 'command', { flags })
+  const output = await new Fixture('./fixtures/command')
+    .withFlags({ testOpts: { errorMonitor: true }, bugsnagKey: BUGSNAG_TEST_KEY })
+    .runWithBuild()
+  t.snapshot(normalizeOutput(output))
 })
 
 test('Report configuration user error', async (t) => {
-  await runFixture(t, 'config', { flags })
+  const output = await new Fixture('./fixtures/config')
+    .withFlags({ testOpts: { errorMonitor: true }, bugsnagKey: BUGSNAG_TEST_KEY })
+    .runWithBuild()
+  t.snapshot(normalizeOutput(output))
 })
 
 test('Report plugin input error', async (t) => {
-  await runFixture(t, 'plugin_input', { flags })
+  const output = await new Fixture('./fixtures/plugin_input')
+    .withFlags({ testOpts: { errorMonitor: true }, bugsnagKey: BUGSNAG_TEST_KEY })
+    .runWithBuild()
+  t.snapshot(normalizeOutput(output))
```

Comment:
It's too bad that ava doesn't allow custom snapshot serializers like jest, where we could do our normalization.